includes/webform.admin.inc

```diff
@@ -250,6 +250,13 @@ function webform_admin_settings() {
     '#description' => t('Tokens can be used to reveal sensitive information. Allow tokens if Webform creators are trusted.'),
   );
 
+  $form['advanced']['webform_allowed_tags'] = array(
+    '#type' => 'textfield',
+    '#title' => t("Allowed HTML tags in labels"),
+    '#default_value' => implode(' ', webform_variable_get('webform_allowed_tags')),
+    '#description' => t('List of the HTML tags allowed in component labels, descriptions, and also the individual labels of select list items. For example, <code>a em strong code img</code>. JavaScript event attributes, JavaScript URLs, and CSS are always stripped.'),
```

Comment:
While we give an example and a default, it might be useful to be explicit that this is a space separated list

```suggestion
    '#description' => t('Space separated list of the HTML tags allowed in component labels, descriptions, and also the individual labels of select list items. For example, <code>a em strong code img</code>. JavaScript event attributes, JavaScript URLs, and CSS are always stripped.'),
```